### PR TITLE
CONCF-1060 | Curated Content Edit Tool — tooltips for form inputs

### PR DESCRIPTION
**Pull request by Warkot**

### CODE REVIEW

File: front/styles/main/app.scss [hunk 1]
```diff
@@ -14,6 +14,7 @@
 @import 'component/index';
 @import 'module/index';
 
-// temporary solution while working on styling
+// temporary solution while these inputs are not a part of the Style Guide
```
Comment (vforge) [inline front/styles/main/app.scss]: :+1:

**Comment by vforge** [comment]
LGTM, are fail in jenkins relevant?

**Comment by Warkot** [comment]
Mercury-Server failing because of this would be surprising. :wink: 
```
[16:40:22] Finished 'node-test' after 9.23 ms

/var/lib/jenkins/workspace/Mercury-Server/93/test/node-qunit.runner.js:49
		throw err;
		      ^
Error: Process blocked for too long
    at null.<anonymous> (/var/lib/jenkins/workspace/Mercury-Server/93/node_modules/qunit/lib/testrunner.js:83:30)
    at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)

/var/lib/jenkins/workspace/Mercury-Server/93/gulp/tasks/node-test.js:22
				throw 'Tests failed';
				^
Tests failed
+ echo BUILD:FAILURE
```

**Comment by vforge** [comment]
Oks :)

**Comment by bognix** [comment]
LGTM

**Pull request merged**